spec.html

```diff
@@ -10,7 +10,7 @@
   }
 </script>
 <pre class=metadata>
-  title: ECMAScript® 2016 Language Specification
```

Comment:
I don't much care whether the spec represents non-ASCII characters with literal characters or entity references, but I think it should be consistent, and it was a much smaller diff to change literals to entity refs than vice versa. If @bterlson would prefer vice versa, I can commit that too.

(As for changing literals to entity refs during the build process, I don't think the build process has any need to do that.)